### ISSUE TITLE
Do not translate languages in language picker

### ISSUE THREAD

**Issue by Entze** [klimeryk/recalendar.js#20]
Maybe you thought about this in particular already so I might be wrong, but consider this user story (tl;dr further down):

I am a French-speaking user that does not speak Polish and only little English. Visiting recalendar.me and after clicking on 'Create your calendar', I see a language picker. I want the interface and the calendar to be in my locale so I try to pick 'French'. However I accidentally click on 'Polish'. I click the language picker again and see 'Angielski', 'Francuski', 'Hiszpańki', 'Polski'. Granted it is not very difficult to figure out that 'Francuski' would be the correct choice, but shouldn't the language itself not be translated? Alternatively providing visual cues such as flags (but those are opinionated).

tl;dr
Consider not translating the languages in the language picker and additionally providing visual cues such as country-flags.

**Comment by klimeryk** [comment]
Thanks for raising this issue, @Entze. It's a very good question. My thinking here was:
 - I'm using [the language detector from i18next](https://github.com/i18next/i18next-browser-languageDetector). If ReCalendar has translations for that language, it should be automatically activated.
 - Otherwise, English is used as fallback.

However, this makes the assumption that everyone knows English. Which is probably true for a website like this, but it's also not a good assumption to perpetuate. So I agree that it just as I should not expect anyone to know what their language is in Polish, I should not expect them to know what it is in English. This seems to match [the best practices](https://ux.stackexchange.com/a/37025/45864) (that I should have researched earlier!).

I'll adjust the language picker so that it always will show the language as it's written in that language. So the list would look like:
 - English
 - Polski
 - Norsk
 - Français

It's a win-win, cause it will also make adding languages easier ;)
